src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/Descriptors.cs

```diff
@@ -122,7 +122,14 @@ internal class Descriptors
             nameof(AZC0018),
             "Do ensure protocol method take a RequestContext parameter called context and not take models as parameter type or return type.",
             "Protocol method should have requestContext as the last parameter and don't have model as parameter type or return type.",
-            "Usage", DiagnosticSeverity.Warning, isEnabledByDefault: true, description: null);
+            "Usage", DiagnosticSeverity.Warning, isEnabledByDefault: true, description: null);
+
+        public static DiagnosticDescriptor AZC0020 = new DiagnosticDescriptor(

```

Comment:
This analyzer number skips AZC0019 to leave space for it to be added in this PR: https://github.com/Azure/azure-sdk-tools/pull/6496, as it is closely related to AZC0018.